java/src/main/java/ai/rapids/cudf/ColumnView.java

```diff
@@ -2846,10 +2846,23 @@ public final ColumnVector stringReplace(Scalar target, Scalar replace) {
    * @param repl The string scalar to replace for each pattern match.
    * @return A new column vector containing the string results.
    */
+  @Deprecated
   public final ColumnVector replaceRegex(String pattern, Scalar repl) {
     return replaceRegex(pattern, repl, -1);
```

Comment:
Nit: It's nice when deprecated methods are shown implementing what the caller should have done rather than call other deprecated methods.  Not must-fix by any means.
```suggestion
    return replaceRegex(new RegexProgram(pattern, CaptureGroups.NON_CAPTURE), repl);
```

Reply:
I see. Thank you for pointing it out. I will update this.